khmer/khmer_args.py

```diff
@@ -360,12 +361,15 @@ def build_graph_args(descr=None, epilog=None, parser=None):
     group = parser.add_mutually_exclusive_group()
     group.add_argument('--max-tablesize', '-x', type=float,
                        default=DEFAULT_MAX_TABLESIZE,
-                       help='upper bound on tablesize to use; overrides ' +
+                       help='upper bound on tablesize to use; overrides '
                        '--max-memory-usage/-M')
```

Comment:
'make pep8' checks against our chosen style standard; if it passes that,
we're happy :)